lib/canary/plugs.ex

```diff
@@ -37,10 +37,15 @@ defmodule Canary.Plugs do
   If the resource cannot be fetched, `conn.assigns.resource_name` is set
   to nil.
 
-  If the action is `:index`, all records from the specified model will be loaded.
+  The `:persisted` key can override how a resource is loaded and can be useful when dealing
+  with nested resources.
+
+  If the action is `:index`, all records from the specified model will be loaded. This can
```

Comment:
Super minor thing here, but I think this paragraph might be more clear if the first sentence began "By default, when the action is ...".